apps/dashboard/src/hooks/useClipboard.ts

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useEffect, useState } from "react";
+
+/**
+ * todo: this hook is copied from the SDK, so might as well expose it from the SDK
+ * and use it here?
+ */
```

Comment:
we can consider that but it feels very "random utility"-esque - this is fine for now